src/layer/poi.js

```diff
@@ -7,6 +7,7 @@ var iconDefs = {
   hospital: "hospital",
   medical: ["doctors", "clinic"],
   parking: "parking",
+  school: "school",
```

Comment:
Kindergartens are unlikely to have very distinct icons from other schools, so we could match `kindergarten` at the same time.

Reply:
Kindergartens are now included